app/assets/src/components/views/compare/samples_heatmap_vis.scss

```diff
@@ -61,4 +64,46 @@
       }
     }
   }
+
+  .bannerContainer {
+    width: 500px;
+    bottom: $space-s;
+    left: 50%;
+    margin-left: -250px;
+    text-align: center;
+    position: fixed;
+    transition: opacity 0.5s ease;
+
+    &.show {
+      opacity: 1;
+    }
+
+    &.hide {
+      opacity: 0;
+    }
+
+    &:hover {
+      opacity: 1;
+    }
+
+    .bannerText {
+      @include font-body-xxs;
```

Comment:
We're switching to the `crop` version of these typography mixins. Can you use font-body-xxs-crop?